app/Console/Kernel.php

```diff
@@ -15,7 +15,7 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-         $schedule->command('ChooseRandomVictim')->dailyAt('12:15');
+         $schedule->command('ChooseRandomVictim')->dailyAt(env('CHOOSE_RUNNER_AT', '12:15'));
```

Comment:
Do **NOT EVER** use `env()` in code.

Also, can be called by doing `ChooseRandomVictim::class`